handsontable/src/plugins/dropdownMenu/dropdownMenu.js

```diff
@@ -4,7 +4,7 @@ import { objectEach } from '../../helpers/object';
 import CommandExecutor from '../contextMenu/commandExecutor';
```

Comment:
I'd copy-paste the `handsontable/src/plugins/contextMenu/__tests__/a11y/attributes.spec.js` file to the dropdown menu to make sure that after changing the Menu implementation (or replacement) the plugin will be still covered with the tests.